.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
Nitpick: Should we use `kebab-case` for folder and file names (to be consistent with most of the repo)? 
Sorry, this is mostly a "me" problem. 😁 
So `ISSUE_TEMPLATE/bug_report.md` would be `issue-template/bug-report.md`.

I do realize that the existing pull-request template uses `snake_case`, but we can change that to `kebab-case` later:
https://github.com/GoogleCloudPlatform/bank-of-anthos/blob/master/.github/pull_request_template.md

Reply:
Sweet! Thanks, Christine.
I just realized that the all uppercase `ISSUE_TEMPLATE` is a [GitHub thing](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms).